library/quality/quality.gradle

```diff
@@ -69,8 +69,7 @@ task pmd(type: Pmd) {
 
 android {
     lintOptions {
-        disable 'InvalidPackage',
-                'NewerVersionAvailable' // Comment this line out to do a quick check for library version updates.
+        disable 'InvalidPackage', 'RestrictedApi' // TODO remove RestrictedApi
```

Comment:
I noticed this as well in another project of mine.  I think it's a bug in the gradle plugin alpha 3.  Should we hold off on that rather than ignore these errors?  That would be a good class of errors to not ignore in the future.